apps/crn-frontend/src/events/api.ts

```diff
@@ -1,14 +1,29 @@
 import { AlgoliaSearchClient } from '@asap-hub/algolia';
-import { EventResponse, ListEventResponse } from '@asap-hub/model';
 import { createSentryHeaders } from '@asap-hub/frontend-utils';
-
+import { EventResponse, ListEventResponse } from '@asap-hub/model';
 import { API_BASE_URL } from '../config';
-import { GetEventListOptions } from './options';
 import createListApiUrl from '../CreateListApiUrl';
+import { GetEventListOptions } from './options';
+
+export function getFilters(filters: string[], userId?: string) {
```

Comment:
I am not sure i like this. I dont think we should be constructing queries that aren't tokenized incase someone puts a ) or " or something. Shouldn't this be in the algolia library @peterstarling ?

Reply:
I can't find anything in the Algolia library to suggest this: https://www.algolia.com/doc/api-reference/api-parameters/filters/